traffic_ops_db/test/docker/Dockerfile-db-admin

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+############################################################
+# Dockerfile to build Traffic Ops DB admin test environment
+############################################################
+
+
+FROM centos:7
+
+RUN yum install -y \
+    epel-release \
+    https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-redhat96-9.6-3.noarch.rpm \
+    git \
+    golang
+
+# Override TRAFFIC_OPS_RPM arg to use a different one using --build-arg TRAFFIC_OPS_RPM=...  Can be local file or http://...
+ARG TRAFFIC_OPS_RPM=traffic_ops.rpm
+ADD $TRAFFIC_OPS_RPM /
+RUN yum install -y \
+        /$(basename $TRAFFIC_OPS_RPM) \
```

Comment:
current version on master doesn't work because the `golang` packages were removed from centos base for 7.6.
As a workaround,  need to use `yum-config-manager --add-repo ...` as is done in `infrastructure/docker/build/Dockerfile-traffic_ops`

This will need to be removed once we add the new location for the `golang` packages (and upgrade to 1.11)

Reply:
thanks, fixed